packages/polaris-viz/src/components/ChartElements/ChartDiv.tsx

```diff
@@ -1,16 +1,17 @@
-import type {ReactNode} from 'react';
+import type {CSSProperties, ReactNode} from 'react';
 
 import styles from './ChartElements.scss';
 
 interface ChartDivProps {
-  height: number;
-  width: number;
+  height?: number;
```

Comment:
Sorry, can we keep these are required? I don't think we want to rely on the style tag to set height/width